src/internet_identity/tests/integration/http.rs

```diff
@@ -637,33 +637,23 @@ fn should_list_aggregated_session_seconds_and_event_data_counters() -> Result<()
     let metrics = get_metrics(&env, canister_id);
     assert_metric(
         &metrics,
-        "internet_identity_prepare_delegation_session_seconds{dapp=\"https://some-dapp-1.com\",window=\"24h\",ii_origin=\"other\"}",
-        5400f64,
-    );
-    assert_metric(
-        &metrics,
-        "internet_identity_prepare_delegation_session_seconds{dapp=\"https://some-dapp-1.com\",window=\"30d\",ii_origin=\"other\"}",
+        "internet_identity_prepare_delegation_session_seconds{dapp=\"https://some-dapp-2.com\",window=\"24h\",ii_origin=\"ic0.app\"}",
         5400f64,
```

Comment:
Sorry, but why is it 5400?

I see that the session lasts 3600 one time and None the second.

Reply:
Because `None` defaults to 1800 seconds (30 minutes). I can also set an explicit value if you prefer.

Reply:
Ah, ok, got it.